.travis.yml

```diff
@@ -12,13 +12,10 @@ branches:
     - travis
 
 rvm:
-  - 1.8.7
```

Comment:
Hmm, I thought these were failing on Master. I see the unit tests are passing here: https://travis-ci.org/elastic/elasticsearch-ruby/builds/120013643 . I must have been mistaken